pkgs/applications/networking/twingate/default.nix

```
@@ -0,0 +1,52 @@
+{ autoPatchelfHook
+, curl
+, dpkg
+, dbus
+, fetchurl
+, lib
+, libnl
+, udev
+, cryptsetup
+, stdenv
+, pkgs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "twingate";
+  version = "1.0.60";
+
+  src = fetchurl {
+    url = "https://binaries.twingate.com/client/linux/DEB/${version}/twingate-amd64.deb";
+    sha256 = "b308c422af8a33ecd58e21a10a72c353351a189df67006e38d1ec029a93d5678";
+  };
+
+  buildInputs = [ curl libnl udev cryptsetup ];
+  nativeBuildInputs = [ dbus dpkg autoPatchelfHook ];
```

```suggestion
  nativeBuildInputs = [ dbus dpkg autoPatchelfHook ];

  buildInputs = [ curl libnl udev cryptsetup ];
```
Why do we need dbus in nativeBuildInputs?